src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -23,6 +23,10 @@ RoboCup Bangkok was the first in-person competition in three years due to COVID-
 - Long ethernet cables became tedious to use and untangle
 - Heat dissipation is an issue
 
+#### Motions
+
+- Rollover from back to front, then using get up front does require a lot of floor space and caused a few tangles, such as the robot getting its feet caught in the goal net.
```

Comment:
You should look into bitbots solution using splines for dynamic getups, try to move away from scripts entirely